toonz/sources/toonzlib/fullcolorpalette.cpp

```diff
@@ -43,10 +44,15 @@ TPalette *FullColorPalette::getPalette(ToonzScene *scene) {
   m_palette->addRef();
   TFilePath fullPath = scene->decodeFilePath(m_fullcolorPalettePath);
   if (!TSystem::doesExistFileOrLevel(fullPath)) {
-    // Per I francesi che hanno il nome vecchio della paletta
-    // Verra' caricata la vecchia ma salvata col nome nuovo!
+    // For the French who have the old name of the headstock
```

Comment:
Great comment DV?

Reply:
lol. I Google translated it like I normally do when I come across non-english comments and paste it as is. Sometimes it makes sense. :)

Would you like me to revert it?